scanapp/models.py

```diff
@@ -129,39 +98,43 @@ def __str__(self):
 
 
 class MatchedRule(models.Model):
```

Comment:
I think that for simplicity you can use a JSONField at the License level to store a matched rule... and no specific models